bot/cogs/sticky.py

```diff
@@ -36,10 +36,10 @@ async def on_message(self, message: discord.Message) -> None:
         if res and message.author != self.bot.user:
             try:
                 sticky = await message.channel.fetch_message(sticky_message_id)
-            except discord.errors.NotFound:  # This is happened when the chat is going fast
-                return
+                await sticky.delete()
+            except discord.errors.NotFound:
+                pass
```

Comment:
![image](https://user-images.githubusercontent.com/20255031/220009203-b10238b5-44be-4ac3-a530-83501020bfdd.png)

This logic is failed on fast on-going message